refinery/ui/source/js/dashboard/controllers/data-sets-card-ctrl.js

```diff
@@ -229,13 +232,20 @@
     function searchDataSets (query) {
       // reset perm filter until we can search & check perms
       vm.groupFilter = { selectedName: 'All' };
+      vm.params = { limit: vm.itemsPerPage, offset: vm.pageStartOffset };
       if (query && query.length > 1) {
         vm.loadingDataSets = true;
         var apiRequest = new DataSetSearchApi(query);
-        apiRequest(200).then(function (response) {
+        apiRequest(100).then(function (response) {
```

Comment:
Not understanding the need for a `100` instead of a `200` here

Reply:
Oops, good catch. It's the number of ds returned.